src/python/README.rst

```diff
@@ -17,7 +17,7 @@ Turi Create supports:
 System Requirements
 -------------------
 
--  Python 2.7, 3.5, or 3.6
+-  Python 2.7, 3.5, 3.6, or 3.7 (macOS only)
```

Comment:
We might want to clarify that Linux is supported on 2.7, 3.5, and 3.6? To me it looks like all of them are macOS only the way it's written.